build.sbt

```diff
@@ -68,7 +68,21 @@ libraryDependencies ++= Seq(
   "org.specs2" % "specs2_2.11" % "2.3.13" % "test"
 )
 
-scalacOptions ++= Seq("-unchecked", "-deprecation")
+// The OpenIE codebase generates compiler warnings like this:
+//
+//   Adapting argument list by creating a 2-tuple
+//   Adapting argument list by creating a 3-tuple
+//   implicit conversion method ... should be enabled
+//   method deserialize in object ... is deprecated
+//   possible missing interpolator
+//   reflective access of structural type member method close should be enabled
+//   side-effecting nullary methods are discouraged
+//   type parameter String defined in method apply shadows type String defined in object Predef. You may want to rename your type parameter, or possibly remove it.
+//
+// The code should be fixed to not have these problems. Until then, "-nowarn"
```

Comment:
Here's what it looks like without -nowarn:

https://gist.github.com/aimichal/54d5f67c84d34551cbecae1452116139

My concern is that running the code with "sbt runMain edu.knowitall..." will produce this noise. That only happens the first time through the compile though, so maybe it's not as problematic as I expect.